app/src/main/java/com/markkolenbrander/capstonenewsapp/models/Article.kt

```diff
@@ -1,17 +1,18 @@
 package com.markkolenbrander.capstonenewsapp.models
 
 import android.os.Parcelable
+import com.squareup.moshi.Json
 import kotlinx.parcelize.Parcelize
 
 @Parcelize
 data class Article(
-    val source: Source,
-    val author: String? = null,
-    val title: String,
-    val description: String? = null,
-    val url: String,
-    val urlToImage: String? = null,
-    val publishedAt: String,
-    val content: String,
+    @Json(name = "source") val source: Source?,
+    @Json(name = "author")val author: String? = null,
+    @Json(name = "title")val title: String? = null,
+    @Json(name = "description")val description: String? = null,
+    @Json(name = "url")val url: String? = null,
+    @Json(name = "urlToImage")val urlToImage: String? = null,
+    @Json(name = "publishedAt")val publishedAt: String? = null,
+    @Json(name = "content")val content: String? = null,
```

Comment:
Great job adding the @Json annotations.  Just note, you don't have to add the annotations as by default Gson will attempt to convert the properties based on the variable name.  Typically, the @Json annotation is needed for when variable names differ from their json property counterparts.

Reply:
Ok, thanks. I thought because in the video they said for best practice and not taking risks to use annotations for al the properties, eventhough the names are are the same